editor/editor_help.cpp

```diff
@@ -1612,6 +1612,14 @@ void EditorHelp::generate_doc() {
 	doc->merge_from(compdoc); //ensure all is up to date
 }
 
+void EditorHelp::_toggle_scripts_pressed() {
+	if (is_layout_rtl()) {
+		toggle_scripts_button->set_icon(ScriptEditor::get_singleton()->toggle_scripts_panel() ? get_theme_icon(SNAME("Forward"), SNAME("EditorIcons")) : get_theme_icon(SNAME("Back"), SNAME("EditorIcons")));
+	} else {
+		toggle_scripts_button->set_icon(ScriptEditor::get_singleton()->toggle_scripts_panel() ? get_theme_icon(SNAME("Back"), SNAME("EditorIcons")) : get_theme_icon(SNAME("Forward"), SNAME("EditorIcons")));
```

Comment:
Can be a little less repetitive:
```suggestion
		toggle_scripts_button->set_icon(get_theme_icon(ScriptEditor::get_singleton()->toggle_scripts_panel() ? SNAME("Forward") : SNAME("Back"), SNAME("EditorIcons")));
	} else {
		toggle_scripts_button->set_icon(get_theme_icon(ScriptEditor::get_singleton()->toggle_scripts_panel() ? SNAME("Back") : SNAME("Forward"), SNAME("EditorIcons")));
```
(same for the other similar parts)